### ISSUE TITLE
[CoE Starter Kit - BUG] Deleting envts from Power Platform Admin View does not delete the object from the tenant

### ISSUE THREAD

**Issue by Raz-21** [microsoft/coe-starter-kit#2944]
### Describe the issue

At any time when we delete the instance from Power Platform admin center, this will not delete from COE and this issue is persistent, another issue if you deploy an instance from Power Platform admin center, it won't appear or listed in COE environment, happen only once and don't know what action needs to take in order to get missing instance appears in to in COE list of environment.

Regards

### Expected Behavior

Technically at any time when instance will be rename or delete it, COE environment list must reflect with updated info which is replica of your Power Platform admin center i.e. Environment list must match what you see in admin center same as for COE

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

3.38

### What app or flow are you having the issue with?

not sure

### Steps To Reproduce

Go to Power Platform
delete any instance or rename any instance
go to COE
look for deleted environment under environment section

**Comment by Jenefer-Monroe** [comment]
Sorry I dont follow, what do you mean by instance?

**Comment by Raz-21** [comment]
> Sorry I dont follow, what do you mean by instance?


Actually What i am referring to instance = Sandbox

**Comment by Jenefer-Monroe** [comment]
So you are deleting an environment? From this UX? Is that what you mean? 
![image](https://user-images.githubusercontent.com/89584709/172718426-41f16cb9-eb10-422e-88f1-0fa29d779095.png)

**Comment by Raz-21** [comment]
> So you are deleting an environment? From this UX? Is that what you mean? ![image](https://user-images.githubusercontent.com/89584709/172718426-41f16cb9-eb10-422e-88f1-0fa29d779095.png)

That's correct,

**Comment by Jenefer-Monroe** [comment]
Gotcha. 

This UX is just a view of the tenant inventory. Deleting here only deletes the object from inventory, it does not delete the object from the tenant. 

To manually delete an environment from the tenant you should use the product UX: [Delete envionrment](https://docs.microsoft.com/power-platform/admin/delete-environment)

And then the **_CLEANUP - Admin | Sync Template v3 (Check Deleted)_** flow will detect it has been deleted and update the inventory when it runs. 

Hope that clears things up for you.

**Comment by Raz-21** [comment]
Thanks Monroe
My apologies for a late response. You response does make sense to me though I saw check deleted flow was not running and having some issue. Let me see to get this fix and once done I will keep you posted.

Thank you again.

Raza
________________________________

**Comment by Jenefer-Monroe** [comment]
You bet, please go through the [inventory setup](https://docs.microsoft.com/power-platform/guidance/coe/setup-core-components) instructions again and be sure that all the flows are turned on. Without all of them on and running, your inventory will not be complete and clean.

**Comment by Jenefer-Monroe** [comment]
Closing out as no further action for starter kit team
